go.mod

```diff
@@ -54,6 +54,7 @@ require (
 	k8s.io/apiextensions-apiserver v0.18.5
 	k8s.io/apimachinery v0.18.5
 	k8s.io/client-go v0.18.5
+	k8s.io/klog v1.0.0
```

Comment:
Do not use klog. We have moved to the uber zap logging framework.

Reply:
Done